docs/versioned_docs/version-6.1.x/configuration/overview.md

```diff
@@ -7,7 +7,62 @@ title: Overview
 
 ### Generating a Cookie Secret
 
-To generate a strong cookie secret use `python -c 'import os,base64; print(base64.urlsafe_b64encode(os.urandom(16)).decode())'`
+To generate a strong cookie secret use one of the below commands:
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs
+  defaultValue="python"
+  values={[
+    {label: 'Python', value: 'python'},
+    {label: 'Bash', value: 'bash'},
+    {label: 'OpenSSL', value: 'openssl'},
+    {label: 'PowerShell', value: 'powershell'},
+    {label: 'Terraform', value: 'terraform'},
+  ]}>
+  <TabItem value="python">
+
+  ```shell
+  python -c 'import os,base64; print(base64.urlsafe_b64encode(os.urandom(32)).decode())'
+  ```
+
+  </TabItem>
+  <TabItem value="bash">
+
+  ```shell
+  cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1 | base64
```

Comment:
From my ARM Mac (not sure if different from older Darwin/x86_64 Macs), this one didn't work:

```
cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1 | base64
tr: Illegal byte sequence
```

Reply:
Interesting, not sure why it would do that. I'll see if I can reproduce on my Pi, it's about the only arm I have access to